### PR TITLE
added game board to portfolio website

### PR DESCRIPTION
**Pull request by prra-codes**
Title: Add game board to portfolio website.

Summary: Added game board to portfolio website. This pull request is just to show the Connect 4 game code has been imported and to examine the generic layout for the portfolio page now: Profile image and text on the left, game board on the right and contact details at the bottom. I am thinking that the left side (portfolio image) should take up 50% of the website width and the right side with the game board should take the other 50%. I have also pinned (a while ago) my 6 main projects on my github page so that is the first thing others will see when they click my github link in the contact info.

Test Plan: Honestly speaking, a lot of CSS needs to change. The game board is too big, the text overlays on the game board, I don't have the header anymore (for the game board), the game board moves a lot when a player wins now that I've imported the code, etc. I've gotten rid a lot of text  but more CSS need to go and be readjusted. Some of my portfolio website css is interfering with the text css of the game board. I will fix all of these issues for the next pull request. Again, this pull request is to ask if the layout idea is good before I strive to made it look super good and fix the aforementioned issues!

### CODE REVIEW

**Comment by jacksongabbard** [comment]
So, your test plan doesn't actually help me if I want to know how your stuff looks / works. What commands did you run to test it? A good test plan is something like:

```
Test plan: 
- Loaded the site locally by running `npm run start` and spinning up the create react app page
- Poked around a noticed a bunch of issues:
  - The game board is too big
  - the text overlays on the game board
  -  the header is gone
  - etc.
The goal here isn't a beautiful site. This is just a first pass at changing over the homepage.
```

Your commit message shouldn't start with the word `Title:`. When you run `git commit` and edit the text file, whatever text you put on the first line becomes the title. Github will pick that up automatically. You don't need to prefix it with `Title: `. 

Your summary shouldn't just restate the title. Basically, you can skip the first sentence of your summary and start with `Summary: This pull request is just...`.

File: src/components/Board/Board.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  background-color: blue;
+  border: 10px solid navy;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  /* cursor: pointer; */
+
+  /* cursor: url("../../images/actual-red-disc.png"), auto; */
+  cursor: url("../../images/actual-yellow-disc.png"), auto;
```
Comment (jacksongabbard) [inline src/components/Board/Board.css:14]: You should always alphabetise your CSS. Also, don't leave commented out code in your files:

```
  background-color: blue;
  border: 10px solid navy; 
  cursor: url("../../images/actual-yellow-disc.png"), auto;
  display: flex;
  flex-wrap: wrap;
  height: 540px;
  margin: 0 auto;
  width: 630px;
```

File: src/components/Board/Board.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+.new-game {
+  height: 30px;
+  width: 100px;
+  background-color: #0362fc;
+  border: none;
+  border-radius: 6px;
+  color: white;
+  font-weight: bold;
+  cursor: pointer;
```
Comment (jacksongabbard) [inline src/components/Board/Board.css:38]:
```
  cursor: pointer;
  background-color: #0362fc;
  border: none;
  border-radius: 6px;
  color: white;
  font-weight: bold;
  height: 30px;
  width: 100px;
```

File: src/components/Tile/Tile.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.tile {
+  height: 70px;
+  width: 70px;
+  margin: 5px;
+  background-color: white;
+  border-radius: 50%;
+  border: 5px solid navy;
+  /* cursor: pointer; */
+  /* cursor: url(redDisc); */
+  /* cursor: url("../../images/red_disc.svg"); */
```
Comment (jacksongabbard) [inline src/components/Tile/Tile.css:10]: Don't leave commented out code in your files.

File: src/index.css [hunk 4]
```diff
@@ -31,17 +31,36 @@
   background: var(--clr-dark);
 }
 
+.testing-one {
+  /* background-color: green; */
+}
+
+.testing-two {
+  /* background-color: yellow; */
+}
```
Comment (jacksongabbard) [inline src/index.css:40]: These look like unused code. It's bad coding hygiene to leave things like this in your files.

File: src/index.css [hunk 5]
```diff
@@ -31,17 +31,36 @@
...
-.max-width-870 {
-  max-width: 870px;
+/* .max-width-50-percent { */
+/* max-width: 870px; */
+/* max-width: 50%; */
+/* } */
+
+.intro-width {
+  /* max-width: 50%; */
+  /* min-width: 500px; */
```
Comment (jacksongabbard) [inline src/index.css:58]: Can you guess what I'm going to say? 😉

File: src/index.css [hunk 6]
```diff
@@ -96,6 +115,7 @@ body {
 
 .section-intro {
   padding: 4em 2em;
+  /* background-color: red; */
```
Comment (jacksongabbard) [inline src/index.css:118]: same

File: src/index.css [hunk 7]
```diff
@@ -110,7 +130,8 @@ body {
 
 img {
   display: block;
-  max-width: 100%;
+  /* max-width: 100%; */
```
Comment (jacksongabbard) [inline src/index.css:133]: same

File: src/index.css [hunk 8]
```diff
@@ -226,6 +248,12 @@ h3 {
   border-radius: 50%;
 }
 
+.center-img {
+  display: flex;
+  /* background-color: red; */
```
Comment (jacksongabbard) [inline src/index.css:253]: same

**Review by jacksongabbard — CHANGES_REQUESTED** [review]
Good to see this work coming through. There's a lot about this that could use a pick-me-up. Btw, I'm reviewing your code right now from the 109 bus between errands.

File: src/components/Board/Board.css (hunk 1 above)
Comment (prra-codes) [reply src/components/Board/Board.css:14]: Thank you, I'll adjust it.

File: src/components/Tile/Tile.css (hunk 3 above)
Comment (prra-codes) [reply src/components/Tile/Tile.css:10]: I'll get rid of all the uncommented code

File: src/index.css (hunk 4 above)
Comment (prra-codes) [reply src/index.css:40]: I was testing some stuff with flexbox and getting the text and images on the left and the main board on the right. I'll delete it.

File: src/index.css (hunk 5 above)
Comment (prra-codes) [reply src/index.css:58]: I'm sorry! I'll get rid of all the unnecessary comments

**Comment by prra-codes** [comment]
Thanks for all the advice, gonna implement it and then make another pull request with the corrections

**Pull request closed without merging**